test.md

```diff
@@ -0,0 +1 @@
+This will be ignored by .gitignore file
```

Comment:
Kindly add the test.md file to the gitignore file and make sure it is actually ignored by git since it is part of the project requirement.